syntax.php

```diff
@@ -83,7 +83,8 @@ public function render($mode, Doku_Renderer $renderer, $data) {
         global $lang;
 
         if($mode == 'xhtml') {
-            $disablecache = null;
+            $disablecache = true;
+            if($disablecache) $renderer->info['cache'] = false;
             $namespaceinput = $this->_htmlNamespaceInput($data['namespace'], $disablecache);
```

Comment:
Please note, there was some decision making in the `_htmlNamespaceInput() ` method, which modifies $disablecache by reference.

Reply:
Damn, I failed to consider that, thanks for the heads up @Klap-in.

So in fact this change is not only useless, but also harmful because $renderer->info['cache'] will no longer be set correctly.

I'll revert.